app/views/registration_wizard/choose_school.html.erb

```diff
@@ -17,37 +17,37 @@
       <h1 class="govuk-heading-xl">Choose your school</h1>
 
       <% if @form.display_schools? %>
-        <% if @form.possible_schools.blank? %>
-          <p class="govuk-body">Please choose from schools located in <%= @form.wizard.store["school_location"] %></p>
+        <% if @form.possible_institutions.blank? %>
+          <p class="govuk-body">Please choose from schools located in <%= @form.wizard.store["institution_location"] %></p>
 
           <p class="govuk-body">No schools were found with that name, please try again</p>
 
-          <%= f.govuk_text_field :school_name,
+          <%= f.govuk_text_field :institution_name,
                                  width: "full",
                                  label: { text: "Enter your school name" } %>
```

Comment:
These aren't just schools anymore. Do we plan on changing the copy on this page?

Reply:
Yeah, these will come in once all the backend changes are in